client/src/components/Explorer/Explorer.scss

```diff
@@ -64,11 +62,13 @@ $menu-footer-height: 50px;
     }
 
     // Overrides for default link hover.
+
     &:hover {
```

Comment:
I'm pretty sure this is coming from "newline between rulesets" as you noted later, which I added off of the Wagtail CSS style guide for "Put line breaks between rulesets."
I'll double check. As you say later, it's not so great when there's a comment.